src/logicBoard.js

```diff
@@ -0,0 +1,12 @@
+export function loadScoreBoard(arr) {
+  const ulScore = document.getElementById('ulScore');
+  ulScore.classList = 'list-group';
+  for (let i = 0; i < arr.length; i += 1) {
+    const lsScore = document.createElement('li');
+    lsScore.classList = 'list-group-item';
+    lsScore.innerText = `${arr[i].name}: ${arr[i].score}`;
+    ulScore.appendChild(lsScore);
+  }
+}
+
+export default loadScoreBoard;
```

Comment:
- [ ] good job setting up Webpack 👍 however according to the requirements you are supposed to use ES6 modules to write modular JavaScript with proper ES6 syntax which includes usage of arrow functions instead of the function keyword.